pkg/clusteragent/autoscaling/workload/loadstore/workload_status.go

```diff
@@ -0,0 +1,107 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
nit: since we are already in `workload` package thoughts on just renaming the file to `dump.go` to be consistent with autoscaling store naming?